docs/docs/target/container_image.md

```diff
@@ -154,6 +154,8 @@ See https://avd.aquasec.com/misconfig/ds026
 !!! tip
     You can see how each layer is created with `docker history`.
 
+The [AVD-DS-0016](https://avd.aquasec.com/misconfig/dockerfile/general/avd-ds-0016/) check is disabled for this scan type, see [issue](https://github.com/aquasecurity/trivy/issues/7368) for details.
```

Comment:
This text looks nondescript. I assume users will simply skip this information.
Maybe we want to highlight it in a note/warning or make a table with rules disabled?
@knqyf263 wdyt?

Reply:
Making a table sounds like a good idea, but I'm not sure we'll add more disabled checks. A table might be too much just for a single check.
I think it's enough as we're showing the logs, but using tables or bullet points would also be good. I'll leave it to you two.

Reply:
I think it's fine as is. We can revisit it if we have more disabled checks in the future.